Eluant/LuaApi.cs

```diff
@@ -58,7 +58,7 @@ public enum LuaGcOperation : int
             SetStepMul = 7,
         }
 
-        internal const string LUA_DLL = "lua5.1";
+        internal const string LUA_DLL = "lua51.dll";
```

Comment:
Even though this is the Windows DLL name I'd still recommend omitting the extension, just leave it `"lua51"`. The runtime will append the extension automatically.